README.md

```
@@ -1,23 +1,73 @@
 
+# Build for production
+npm run build
+
+# Preview production build
+npm run preview
 ```
-npm install
+
+## Project Structure
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)